static/css/app.css

```diff
@@ -58,6 +58,18 @@ label.warning {
     padding: 0 5px;
     border-radius: 5px;
 }
+
+.callout.warning {
+    margin-right: 14px;
```

Comment:
Can you explain where this 14px comes from?

Reply:
to bring warning label, email label, input box and button in a proper alignment this 14px has been calculated.

Reply:
same on the about page.

Reply:
And looks good and properly aligned.

Reply:
Still, I don't like the solution. I'd rather see a foundation "native" solution, using the grid.